README.md

```diff
@@ -29,6 +33,7 @@ What is the purpose of the `alt` attribute? e.g:
 
 ```text
 Your answer here.
+The `alt` attribute lets the use know what the image content is by providing alternative text.
 ```
```

Comment:
Yes, but to be more specific- if the user is unable to view the image for whatever reason, the alt attribute will display the text of the image to indicate it's content. OR, if a user is blind or visually impaired and requires the use of a screen reader.